magento/pwa-studio
# [bug] Fix w10 issues

## Description
Informed had a [bug](https://github.com/joepuzzo/informed/issues/215), introduced in version 2, that broke edge on w10.

This PR updates our version of informed to the latest minor. Informed has released a major but I didn't want to introduce that much risk in this PR. That said, there are some semi risky changes that involved changing how we interact with the `formApi`. At some point between our informed versions the author changed the root component to be a function component which does not have an `instance`. So our references in tests to `form.instance` breaks. This also removes the ability to pass a `ref` directly. To remedy this I replaced these usages of instance methods with usage of the `getApi` callback.

We also had a problem with spreading a nodelist which is invalid syntax in edge. Converting it to `Array.from` fixes the issue of an infinite loop of rendering which was occurring due to the problematic code living in the wrapping component of `App`.

## Related Issue





Closes [PWA-266](https://jira.corp.magento.com/browse/PWA-266).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Open a PDP on W10/Edge.
2. Change the quantity - the UI should reflect the selection.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
